operator/data/profiles/default.yaml

```diff
@@ -251,6 +244,11 @@ spec:
   # Global values passed through to helm global.yaml.
   # Please keep this in sync with manifests/global.yaml
   values:
+    # You may override parts of meshconfig by uncommenting the following lines.
+    meshConfig:
```

Comment:
why we need to move to values?

Reply:
Well the current top level field is broken. If we fix that I suppose we can undo this, and then the behavior will be identical